### ISSUE TITLE
Ability to archive items

### ISSUE THREAD

**Issue by christianp** [numbas/editor#486]
An archived item doesn't show up in search results, but there should be a way of revealing it when browsing the project it belongs to.

Everyone who currently has access keeps it (including public accessibility). There should be an option to direct users to a newer version of the question.
